src/local_scheduler.rs

```diff
@@ -103,19 +102,14 @@ impl LocalScheduler {
         //TODO accumvalues needs to be done
     ) {
         let result = Some(result);
-        match event_queues.lock().get_mut(&(task.get_run_id())) {
-            Some(queue) => {
-                queue.push_back(CompletionEvent {
-                    task,
-                    reason,
-                    //                    result: Some(Box::new(result)),
-                    result: result,
-                    accum_updates: HashMap::new(),
-                });
-            }
-            None => {
-                //                println!("ignoring completion event for DAG Job");
```

Comment:
For None case, it should be logged whatever inside the println statement. Just replace it with info!